### PR TITLE
Added aggregate cells function to README and vignette

### PR DESCRIPTION
**Pull request by william-hutchison**
Added a description and example of the aggregate cells function to the README and vignette. Also added the aggregate cells function to the utilities list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,11 +49,13 @@ both Bioconductor and tidyverse worlds.
...
-|-----------------|-----------------------------------------------------------------------------------|
-| `tidy`          | Add `tidySingleCellExperiment` invisible layer over a SingleCellExperiment object |
-| `as_tibble`     | Convert cell-wise information to a `tbl_df`                                       |
-| `join_features` | Add feature-wise information, returns a `tbl_df`                                  |
+| Utilities        | Description                                                                       |
+|------------------|-----------------------------------------------------------------------------------|
+| `tidy`           | Add `tidySingleCellExperiment` invisible layer over a SingleCellExperiment object |
+| `as_tibble`      | Convert cell-wise information to a `tbl_df`                                       |
+| `join_features`  | Add feature-wise information, returns a `tbl_df`                                  |
+| `aggregate_cells`| Aggregate expression data within cell groups                                      |
```
Comment (stemangiola) [inline README.md:57]: "Aggregate expression data within cell groups" should read as "Aggregate cell gene-transcription abundance as pseudobulk tissue"

I don't like to use word "expression" when talking about RNA

File: README.md [hunk 2]
```diff
@@ -757,3 +759,32 @@ tidySingleCellExperiment::pbmc_small_nested_interactions
     ## # … with 90 more rows, and abbreviated variable names ¹​receptor, ²​ligand.name,
     ## #   ³​receptor.name, ⁴​destination, ⁵​interaction.type
     ## # ℹ Use `print(n = ...)` to see more rows
+
+#  Aggregating cells 
+
+Sometimes, it is necessary to aggregate the expression data from a group of cells into a single value. For example, when comparing groups of cells across different samples. Cells within a sample are not independent of each other and treating them as such may result in over-optimistic p-values.
```
Comment (stemangiola) [inline README.md:765]: Again replace "expression" with gene-transcript abundance, adjusting the sentence according to sound good.

From "For example, when comparing groups of cells across different samples." to "For example, when comparing groups of cells across different samples with fixed-effect models."

let's drop the final sentence. It underly a quite huge complexity.

File: vignettes/introduction.Rmd [hunk 3]
```diff
@@ -45,6 +45,7 @@ Utilities | Description
 `tidy` | Add `tidySingleCellExperiment` invisible layer over a SingleCellExperiment object
 `as_tibble` | Convert cell-wise information to a `tbl_df`
 `join_features` | Add feature-wise information, returns a `tbl_df`
+`aggregate_cells` | Aggregate expression data within cell groups
```
Comment (stemangiola) [inline vignettes/introduction.Rmd:48]: As per README

File: vignettes/introduction.Rmd [hunk 4]
```diff
@@ -457,6 +458,17 @@ If the dataset was not so small, and interactions could be identified, you would
 tidySingleCellExperiment::pbmc_small_nested_interactions
 ```
 
+#  Aggregating cells 
+
+Sometimes, it is necessary to aggregate the expression data from a group of cells into a single value. For example, when comparing groups of cells across different samples. Cells within a sample are not independent of each other and treating them as such may result in over-optimistic p-values.
```
Comment (stemangiola) [inline vignettes/introduction.Rmd:463]: As per README

**Review by stemangiola — CHANGES_REQUESTED** [review]

**Review by stemangiola — APPROVED** [review]

**Comment by stemangiola** [comment]
@william-hutchison if you want to add this function to tidyseurat would be awesome. (I gave the function before somewhere I believe)

**Comment by william-hutchison** [comment]
@stemangiola No problem, I can definitely do that.

**Pull request merged**